test/powershell/Language/Classes/scripting.Classes.using.tests.ps1

```diff
@@ -339,7 +339,7 @@ using module Foo
         }
     }
 
-        Context 'Side by side' {
+    Context 'Side by side' {
```

Comment:
Happened to notice the indentation of this line was wrong, decided to fix it since I was already in here